### DISCUSSION TITLE
Impossible from a day to another to deploy my site

### DISCUSSION THREAD

**Discussion by thecreazy** [vercel/community#6366] [Help] [answered]
### Summary

Hello, from 3 days I'm not more able to deploy my site, I'm totally not getting the reason why and I tried hard different solutions and log to get the error, but nothing todo.

An example of the last deploy: https://vercel.com/thecreazys-projects/year-of-data-visualization/E22dVMzPLb9qLz63RrWNjNz38UrC

![Screenshot 2024-04-01 at 18 08 26](https://github.com/vercel/community/assets/4126645/ee24958d-9aeb-43b6-91e0-94e34e10f897)

Locally `npm dev` works well

### Example

https://github.com/thecreazy/year-of-data-visualization

### Steps to Reproduce

Don't know what to write here

- locally works, I can run `npm dev` 
- once pushed the code, the deploy fail

**Comment by amyegan** [comment, marked as answer]
Hey @thecreazy. Thanks for sharing this log info. I see a `SIGKILL` and at least one "Out of Memory" (`OOM`) there. These errors can happen when the build is using a lot of memory. There are a couple of things you can use to solve this.
- Add `NODE_OPTIONS="--max-old-space-size=6144"` to your build command.
- Explore what is using excessive memory during the build process using theses tips: https://nextjs.org/docs/app/building-your-application/optimizing/memory-usage

**Reply by thecreazy** [reply]
Thanks @amyegan, I will try exploring it in depth following your docs, but the strange part to me is that if I try to limit the memory (of my local machine) to `--max-old-space-size=6144` it works, but not on vercel

I really don't get the reason why is going to `OOM`

**Reply by thecreazy** [reply]
ok after a lot of try I found the solution disabling the web pack cache (https://nextjs.org/docs/app/building-your-application/optimizing/memory-usage#disable-webpack-cache)

thx for the support!
